src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -48,5 +48,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[15.0,16.0)" DisplayName="Windows PDB reader/writer" />
```

Comment:
No, the compilers do not depend on this Willow VSIX. There are two copies of DSRN - one for the compilers in msbuild, the other for the IDE. The former isn't a separate component, it's included in msbuild component. The latter is a separate component.

We could make another DSRN component for msbuild, if there are scenarios where msbuild is installed without Roslyn and we would care about having the extra 2MB of dlls there.